added sirius

uixdess opened the pull request.

Commit:
added sirius

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
         </z2_plus>
     </manufacturer>
+	
+	<manufacturer id="Xiaomi">
```
manufacturer Xiaomi is already defined, add your device there

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
     </manufacturer>
+	
+	<manufacturer id="Xiaomi">
+        <Sirius>
```
this should be small letter

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
+            <devicename>MI 8 SE</devicename>
+            <filename>crDroidAndroid-9.0-20190615-sirius-v5.5.zip</filename>
+            <buildtype>Weekly</buildtype>
+            <download>DownloadLink</download>
```
download link should be added 
use your own host

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
+            <filename>crDroidAndroid-9.0-20190615-sirius-v5.5.zip</filename>
+            <buildtype>Weekly</buildtype>
+            <download>DownloadLink</download>
+            <changelog>ChangelogLink</changelog>
```
changelog link also

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
+            <gapps>https://opengapps.org</gapps>
+            <forum>http://4pda.ru/forum/index.php?showtopic=920843</forum>
+            <telegram>https://t.me/uixdesss</telegram>
+        </DeviceCode>
```
this should be closing tag of sirius not DeviceCode

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
+            <forum>http://4pda.ru/forum/index.php?showtopic=920843</forum>
+            <telegram>https://t.me/uixdesss</telegram>
+        </DeviceCode>
+    </manufacturer>
```
this can be removed as Xiaomi is defined already

gwolf2u reviewed (changes requested):
please fix

Commit:
Update update.xml

Commit:
Update update.xml

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
         </z2_plus>
     </manufacturer>
+	
+	<manufacturer id="xiaomi">
```
remove this 
add sirius after jalebi (line 856)

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
+            <filename>crDroidAndroid-9.0-20190615-sirius-v5.5.zip</filename>
+            <buildtype>Weekly</buildtype>
+            <download>https://drive.google.com/file/d/1n2NyBGfgKdF4rLs8EwjpDCClFpwOJmTY/view?usp=drivesdk</download>
+
```
remove blank line

gwolf2u commented on update.xml:
```diff
@@ -889,4 +889,18 @@
+            <gapps>https://opengapps.org</gapps>
+            <forum>http://4pda.ru/forum/index.php?showtopic=920843</forum>
+            <telegram>https://t.me/uixdesss</telegram>
+
```
add closing tag for sirius here

Commit:
Update update.xml

gwolf2u commented on update.xml:
```diff
@@ -865,7 +865,18 @@
         </jalebi>
     </manufacturer>
-
+	
```
Add this before closing manufacturer tag

gwolf2u commented on update.xml:
```diff
@@ -889,4 +900,6 @@
             <telegram>https://t.me/rasenkai</telegram>
         </z2_plus>
     </manufacturer>
+	
```
Remove those two blank lines

Commit:
Update update.xml

gwolf2u commented on update.xml:
```diff
@@ -889,4 +900,5 @@
             <telegram>https://t.me/rasenkai</telegram>
         </z2_plus>
     </manufacturer>
+
```
Remove blank line

Commit:
Update update.xml

Commit:
Update update.xml

Commit:
Update update.xml

gwolf2u committed:
my corrections

removed unnecessary space after lavender closed tag
added empty changelog tag with empty info (mandatory tag as per https://github.com/crdroidandroid/android_vendor_crDroidOTA/blob/9.0/README.md#12-mandatory-xml-tags)
